Author: Kevin Chevreuil - Kaisen <kaisen@kaisenlinux.org>

--- sslyze-5.0.0.orig/docs/conf.py
+++ sslyze-5.0.0/docs/conf.py
@@ -36,7 +36,7 @@ from sslyze import __version__  # noqa:
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["sphinx.ext.autodoc", "sphinx.ext.napoleon", "sphinx_autodoc_typehints"]
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.napoleon"]
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
